src/components/Table/ListTable/TableAction/QuickFilter.vue

```diff
@@ -0,0 +1,328 @@
+          return item.label !== option.label && item.category !== option.category
+        })
+      }
+      option.active = !option.active
+      this.filtered = this.activeFilters.reduce((acc, item) => {
+        return { ...acc, ...item.filter }
+      }, {})
+      this.$emit('filter', this.filtered)
+    }
+  }
+}
+</script>
+<style lang='scss' scoped>
+.quick-filter {
+  background: white;
+  padding: 10px 10px 10px 20px;
+  margin-bottom: 10px;
+  display: flex;
+  place-content: stretch flex-end;
+  justify-content: center;
+  align-content: stretch;
+  box-shadow: 0 1px 1px 0 rgba(54, 58, 80, .32);
+
+  &.shrink {
+    background: inherit;
+    padding: 0;
+    margin-bottom: 0;
+    box-shadow: none;
+  }
+
+  .quick-filter-wrap {
+    display: inline-block;
+    width: calc(100% - 70px);
+
+    .summary-zone {
+      padding-top: 10px;
+      display: flex;
+      justify-content: space-between;
+    }
+
+    .summary-block {
+      .active {
+        ::v-deep .no-margins .num {
+          color: var(--color-primary);
+
+          &::after {
+            content: "\e720";
+            font-family: element-icons !important;
+            font-size: 13px;
+            line-height: 1;
+          }
+        }
+      }
+    }
+
+    .quick-filter-zone {
+      display: flex;
+      justify-content: flex-start;
+      flex-wrap: wrap; /* 允许 item-zone 换行 */
+      gap: 10px;
+
+      h5 {
+        font-weight: 600;
+        text-transform: uppercase;
+        font-size: 12px;
+        margin-bottom: .5rem;
+        line-height: 1.2;
+        display: inline-block;
+      }
+
+      .item-zone {
+        margin-right: 30px;
+        margin-bottom: 5px;
+      }
+
+      .item {
+        display: inline-block;
+        margin-right: 8px;
+        color: #303133;
+        font-size: 12px;
+        cursor: pointer;
+
+        &::after {
+          content: "";
+          margin-left: 4px;
+          margin-bottom: 2px;
+          vertical-align: middle;
+          width: 1px; /* 分割线宽度 */
+          height: 8px; /* 分割线高度 */
+          background-color: var(--color-icon-primary); /* 分割线颜色 */
+          display: inline-block;
+        }
+
+        &:last-child::after {
+          display: none;
+        }
+
+        i {
+          visibility: hidden;
+          margin-left: -3px;
+        }
+
+        &.active {
+          color: var(--color-primary);
+
+          i {
+            visibility: visible;
+          }
+        }
+
+        &:hover {
+          color: var(--color-primary);
+        }
+      }
+
+      ul {
+        list-style: none outside none;
+        margin-block-start: 0;
+        padding-left: 0;
+      }
+    }
+  }
+}
+
+.filter-options {
+  display: block;
+}
+
+.expand-bar-wrap {
+  margin: auto 0;
+  min-width: 60px;
+
+  .expand-bar {
+    float: right;
+    display: block;
+    cursor: pointer;
+
+    i {
+      padding: 5px;
+
+      &.shrink {
+        transform: rotate(180deg);
+      }
+    }
+  }
+}
+
+</style>
```

Comment:
This code has not shown any inconsistencies nor bugs, but it does contain some stylistic improvements that might make it easier to read and maintain:

1. Use `use strict` at the top before `window`, which will enforce stricter error checking.
2. Change import statements to use ES6 syntax for imports.

Let me know if you need these changes done for specific files instead of globally!

Here is an example on how to do this globally for all js files:

```
// Global variables

if (typeof window!=='undefined') {
  // Strict mode
  global.useStrict = true;
}

global.setDebugMode = function(mode){
    if(typeof console === "object") {
        console.debug = mode ? true: false;
    }
}
```